nnvm/python/nnvm/to_relay.py

```diff
@@ -479,7 +480,14 @@ def to_relay(graph, shape_dict, dtype_dict, params):
                 raise Exception(
                     "nnvm.to_relay: unsupported operator: {0}".format(op_name))
 
-    outputs = [relay_map[nid] for nid in output_ids]
+    outputs = []
+    for i in output_ids:
```

Comment:
```suggestion
    for nid, idx, _ in gidx.output_entries:
```
Then, we can remove `heads` and `output_ids`, and replace `i[0]` and `i[1]` with `nid` and `gid` for better readability.